tools/create-dns-pka.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+## Unsure why this offset is needed and present in make-dns-cert
+TOTAL_LENGTH=6
+
+function usage {
+        echo "Usage: "$0" -k [fingerprint of a key in your keyring] -d [dns record you want to create] -u [url where to fetch the key from]"
```

Comment:
Shouldn't this read:
`        echo "Usage: "$0" -f [fingerprint of a key in your keyring] -d [dns record you want to create] -u [url where to fetch the key from]"`?
At least you're using `-f` both in line 11 and as argument for getopts in line 17

Reply:
correct

Reply:
Looks much better now :)